gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -92,19 +98,33 @@ export class CatalogComponent {
   }
 
   public onFilterSelection(event: string) {
-    this.selectedFilter = event;
+    this.filter = event;
```

Comment:
This will be updated after the navigate.

Reply:
Done